docs/content/administration/config-cheat-sheet.en-us.md

```diff
@@ -467,6 +467,7 @@ relation to port exhaustion.
 
 ## Indexer (`indexer`)
 
+- `INDEXER_DEFAULT_FUZZY`: **true**: If the indexer type supports fuzzy search use it by default. Is used by issue- and repo-indexer alike.
```

Comment:
Maybe help the user and list a few known ones that support it? Or maybe ones that don't?

Reply:
Ah, I actually misread.